src/main/java/TaskManager.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Arrays;
+import java.util.ArrayList;
+
+public class TaskManager {
+    private TaskList tasks;
+
+    public TaskManager() {
+        this.tasks = new TaskList();
+    }
+
+    public boolean parseInput(String userInput) {
+        String[] inputs = userInput.split(" ");
+
+        String keyword = inputs[0];
+
+        switch (keyword) {
+            case "bye":
+                return false;
+            case "list":
+                listTaskList();
+                return true;
+            case "done":
+                String msg = tasks.markComplete(Integer.parseInt(inputs[1]));
+                Duke.printBreak();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("     " + msg);
+                return true;
```

Comment:
Perhaps you could store the boolean in a variable and return it at the end of the method, instead of returning in the middle of the switch statement.